JUST.net/JsonTransformer.cs

```diff
@@ -977,7 +977,7 @@ private object LookInTransformed(object output, string propVal, JToken parentTok
             if (output == null && Context.IsLookInTransformed())
             {
                 JToken tmpContext = Context.Input;
-                Context.Input = parentToken;
+                Context.Input = parentToken.Root;
```

Comment:
``parentToken`` can be null so this will result in NullReference exception then.

Reply:
Hmm, parentToken should always be the field being transformed... previously it was explicitly passed as null, but now it isn't. I don't see where it can be null.